internal/pkg/cli/deploy/deploy.go

```diff
@@ -557,21 +608,37 @@ func (d *jobDeployer) DeployWorkload(in *DeployWorkloadInput) (ActionRecommender
 	return nil, nil
 }
 
-func (d *svcDeployer) deploy(in *DeployWorkloadInput, stackConfigOutput svcStackConfigurationOutput) error {
+func (d *workloadDeployer) generateCloudFormationTemplate(conf stackSerializer) (
+	*GenerateCloudFormationTemplateOutput, error) {
+	tpl, err := conf.Template()
+	if err != nil {
+		return nil, fmt.Errorf("generate stack template: %w", err)
+	}
+	params, err := conf.SerializedParameters()
+	if err != nil {
+		return nil, fmt.Errorf("generate stack template configuration: %w", err)
```

Comment:
was this meant to be ⬇️ ?
```suggestion
		return nil, fmt.Errorf("generate stack template parameters: %w", err)
```